SwiftGen.playground/Sources/SwiftGenStoryboardEnumBuilder.swift

```diff
@@ -4,7 +4,8 @@ import Foundation
 
 public final class SwiftGenStoryboardEnumBuilder {
     private typealias Scene = (storyboardID: String, customClass: String?)
```

Comment:
I explored adding an extra `typealias` but thought it would be weird.

Honestly you suggesting it kinds of validates my first thought. 

Let me add it and commit that.